site-manage/datemanager/tool/src/main/java/org/sakaiproject/datemanager/tool/MainController.java

```diff
@@ -63,11 +64,13 @@ public class MainController {
     @Autowired
     private PreferencesService preferencesService;
 
+    @Autowired
+    private static final ResourceLoader rb = new ResourceLoader("Messages");
+
     @GetMapping(value = {"/", "/index"})
     public String showIndex(@RequestParam(required=false) String code, Model model, HttpServletRequest request, HttpServletResponse response) {
 
-        String userId = sessionManager.getCurrentSessionUserId();
-        final Locale loc = StringUtils.isNotBlank(userId) ? preferencesService.getLocale(userId) : Locale.getDefault();
```

Comment:
Do you know if "private PreferencesService preferencesService;" can be removed and cleanup?